src/api/routes/faucets.ts

```diff
@@ -271,9 +267,9 @@ export function createFaucetRouter(db: PgWriteStore): express.Router {
         let sendSuccess: { txId: string; txRaw: string } | undefined;
         let lastSendError: Error | undefined;
         do {
-          const tx = await generateTx(networks[0], nextNonce, fee);
+          const tx = await generateTx(STX_FAUCET_NETWORKS[0], nextNonce, fee);
```

Comment:
@zone117x I think this is an old bug, even though there's a foreach loop right below, the only transaction generated is for the network with index zero... Do we still need to support multiple networks?